website/static/worker.js

```diff
@@ -93,7 +101,7 @@ function formatCode(text, options) {
     // Multiparser may throw if we haven't loaded the right parser
     // Load it lazily and retry!
     if (e.parser && !parsersLoaded[e.parser]) {
```

Comment:
It seems this is the root cause, it always tries to parse again if failed, and the blinking is caused by the syntax error while we're typing, the same thing also occurred in other multi-parser, e.g. css in js-template, etc.

Reply:
but `parsersLoaded.markdown` should be true after the first attempt?

Reply:
Oh, it seems you forgot to add the version to this check:

(L120)

```diff
-parsersLoaded[actualParser] = true;
+parsersLoaded[parserAndVersion] = true;
```

Reply:
Do'h that was when I was trying to get multiple versions working in the same worker thread, `parserAndVersion` should no longer exist.